src/CMakeLists.txt

```diff
@@ -3,25 +3,42 @@ cmake_minimum_required(VERSION 3.10) # Set the minimum required version of CMake
 project(ExperimentalJsonBuilder) # Name your project
 
 # Set C++ standard to C++17 or higher if needed
-set(CMAKE_CXX_STANDARD 23)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
+#set(CMAKE_CXX_STANDARD 23)
+#set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
Same comment here, I think we should do all of this work once (setting standard, sanitizers and build type).